src/utils/date.ts

```diff
@@ -9,3 +9,11 @@ export const getTodaysDate = (): InputDate => {
     year: "" + now.year
   };
 };
+
+export const convertIsoDateToInputDate = (isoDateString: string): InputDate => {
+  const parsedDate = DateTime.fromISO(isoDateString);
+  const day = parsedDate.day.toString().padStart(2, "0"); // Ensure two-digit day
```

Comment:
Maybe not worth it, but there's a `padWithZero()` function in `update.review.beneficial.owner.individual` which could be moved and reused here...

Reply:
Perhaps not an issue for this PR but something to be considered for the future: a date handling package like [MomentJS](https://momentjs.com/). It's insanely awesome particularly as it's possible to compresses sizeable date handling routines into transparent one-liners (amongst other advantages).

Simplified [Timezone handling](https://momentjs.com/timezone/) is also a huge plus. 

Anyway, something to be looked at in a separate session when we can.

Reply:
moment seems to be in maintenance mode now and considered a legacy project -https://www.npmjs.com/package/moment , so we use one of the recommended alternatives, luxon - https://momentjs.com/docs/#/-project-status/recommendations/
DateTime.fromISO is from the luxon package.

Reply:
Ah, got it! Thanks.

So rather than using string and padding callbacks, can we not just use Luxon's formatting options? Like this:

`DateTime.fromISO(isoDateString).toFormat('dd');`

Reply:
I don't want to re-use the padWithZero, it has issues:
`export const padWithZero = (input: string, maxLength: number, fillString: string): string => {
  if (input && input.length > 1){
    return input;
  }
  return String(input).padStart(maxLength, fillString);
};`

a) it's called padWithZero, yet you have to supply the pad string which doesn't have to be a zero. All usages of this function pass the fillString as "0".
b) if input length > 1 then it doesn't pad, just returns the value. This doesn't add anything as .padStart will add leading fillString until it reaches the max length. So this could in theory fill a 1 length input string with 5 leading zeros, which doesn't make sense as input strings > 1 in length just get returned, and all usages of the function are using maxLength 2, fillString "0".
c) don't know why they are using the String(input) constructor when input is already a string.

So it doesn't seem to add anything to the built in string function padStart that is used inside it.

Reply:
although it seems String(undefined).padStart(2, "0") returns undefined, 
whereas undefined.padStart(2, "0") throws an error

Reply:
> Ah, got it! Thanks.
> 
> So rather than using string and padding callbacks, can we not just use Luxon's formatting options? Like this:
> 
> `DateTime.fromISO(isoDateString).toFormat('dd');`

that looks good, i'll try it out

Reply:
updated this one to use the formatting

Reply:
Nice.